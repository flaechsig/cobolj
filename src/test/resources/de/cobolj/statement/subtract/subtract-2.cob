IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-NUMBER-1 PIC 9(3) VALUE 50.
   01 WS-NUMBER-2 PIC 9(3) VALUE 100.
   01 WS-NUMBER-RESULT PIC 9(4) VALUE 1000.

PROCEDURE DIVISION.
   SUBTRACT 5 5 WS-NUMBER-1 FROM WS-NUMBER-2 GIVING WS-NUMBER-RESULT
   NOT ON SIZE ERROR DISPLAY "KEIN SPEICHERUEBERLAUF".
   DISPLAY WS-NUMBER-2.
   DISPLAY WS-NUMBER-RESULT.
STOP RUN.
