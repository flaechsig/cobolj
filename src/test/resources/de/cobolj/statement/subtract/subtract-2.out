KEIN SPEICHERUEBERLAUF
100
0040
